PaperMC/Paper
# Don't enforce icanhasbukkit default if alias block exists

Fixes #9275 

## Test cases:
### Test 1:
1. Start a fresh server
2. File get created
3. Modify file/delete entry and check if `icanhasbukkit` again there
### Test 2:
1. Start a already created server with that file
2. Delete it 
3. Restart server
4. Check if file again back

Both tests are functional
